src/components/DonationCards/DonationCards.tsx

```diff
@@ -0,0 +1,41 @@
+import styles from './DonationCards.module.css';
+import DonationCard from '../DonationCard/DonationCard';
+
+interface DonationCardData {
+   id: number;
+   title: string;
+   text: string;
+   target: number;
+}
+
+const donationCards: DonationCardData[] = [
+   {
+      id: 1,
+      title: 'Save the Whales',
+      text: 'Help us save endangered whale species around the world. Your contributions make a difference.',
+      target: 5000,
+   },
+   {
+      id: 2,
+      title: 'Support Local Education',
+      text: 'Funds go directly to improving education in underfunded areas, providing books, equipment, and training to underprivileged schools.',
+      target: 3000,
+   },
+];
```

Comment:
Lets put this data in a ```mock``` file, as in theory, this information would sit in some database. We are not checking that but lets make it "behave" like you are getting this data from a file as if it was a database.

Reply:
This is actually happening in another pr